apps/stats/stats-1.py

```diff
@@ -375,6 +405,23 @@ def __printPerformance( self, script, args ) :
 					maxLinesPerMetric = args["maxLinesPerMetric"].value
 				)
 
+	def __printContext( self, script, args ) :
+
+			if self.__contextMonitor is None :
+				return
+
+			print "Contexts :\n"
+
+			stats = self.__contextMonitor.combinedStatistics()
+
+			items = [ ( n, stats.numUniqueValues( n ) ) for n in stats.variableNames() ]
```

Comment:
The existing function isn't, no. I thought about adding an overload for the ContextMonitor, and expect I will at some point, but I wasn't totally sure what form it would take yet.